pyfr/solvers/base/system.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, backend, rallocs, mesh, initsoln, nregs, cfg):
         self.ele_ndofs = [e.neles*e.nupts*e.nvars for e in eles]
         self.ele_shapes = [(e.nupts, e.nvars, e.neles) for e in eles]
 
+        # Get the gradient banks
+        self.ele_grad_banks = list(eles.vect_upts_inb)
```

Comment:
I do not think we need this, we can just call .get() on inb.

Reply:
Done